application/helpers/admin/import_helper.php

```diff
@@ -938,6 +957,12 @@ function XMLImportQuestion($sFullFilePath, $iNewSID, $iNewGID, $options = array(
                 $insertdata[(string) $key] = (string) $value;
             }
             unset($insertdata['id']);
+
+            // Avoid inserting attribute if the language of the question is not in the survey.
```

Comment:
Seems alrteady imported here : https://github.com/LimeSurvey/LimeSurvey/blob/d4365162261f1867a3895bb83232da764e22788b/application/helpers/admin/import_helper.php#L907

Same filter `$aLanguagesSupported` do the job